modules/ossm-rn-fixed-issues.adoc

```diff
@@ -19,9 +19,15 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
 
+* https://issues.redhat.com/browse/OSSM-1099[OSSM-1099]
+Kiali shows `Sorry, there was a problem. Try a refresh or navigate to a different page.`
```

Comment:
The Kiali console displayed a message, `Sorry, there was a problem. Try a refresh or navigate to a different page.`